spec/spec_helper.rb

```diff
@@ -20,6 +20,20 @@
 
 Dir[Rails.root.join('spec/support/**/*.rb')].each { |file| require file }
 
+Capybara.register_driver :chrome do |app|
+  args = %w[no-sandbox disable-gpu]
+  args << (ENV['SHOW'].present? ? 'non-headless' : 'headless')
```

Comment:
not sure about the name `SHOW` it's too generic. Maybe something like `HEADLESS` would be better? and we can make it be true by default (in test env) and if we don't want it we do `HEADLESS=false bundle exec rspec`